cli/boards.go

```diff
@@ -77,6 +80,37 @@ func UploadBoardDefsAction(ctx *cli.Context) error {
 	return nil
 }
 
+// DownloadBoardDefsAction is the corresponding action for "board download".
+func DownloadBoardDefsAction(c *cli.Context) error {
```

Comment:
[lowprio] I don't see an obvious reason to export this function. Benji has been trying to not accidentally export things from the cli

Reply:
All the other Action functions for other commands are exported so I was following that convention. Will unexport any other functions tho.